packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/pubspec.yaml

```diff
@@ -21,6 +20,7 @@ dependencies:
     sdk: flutter
   flutter_rx_bloc: ^5.0.1
   flutter_secure_storage: ^7.0.1
+  go_router: 6.0.1
```

Comment:
Please upgrade to the latest rx_bloc packages as well

Reply:
Done